src/main/java/com/playmonumenta/networkchat/channel/Channel.java

```diff
@@ -26,15 +26,18 @@ public abstract class Channel {
 	protected final UUID mId;
 	protected Instant mLastUpdate;
 	protected String mName;
+	protected String mDescription;
+	protected static String mUnsetDescription = "<gray>Description unset.";
 	protected @Nullable TextColor mMessageColor = null;
 	protected ChannelSettings mDefaultSettings;
 	protected ChannelAccess mDefaultAccess;
 	protected final Map<UUID, ChannelAccess> mPlayerAccess;
 
-	protected Channel(UUID channelId, Instant lastUpdate, String name) {
+	protected Channel(UUID channelId, Instant lastUpdate, String name, String description) {
```

Comment:
I wouldn't go changing this constructor actually - better to leave descriptions as a separate, optional method than to modify the existing API/commands to create a new channel.